xarray/plot/plot.py

```diff
@@ -741,6 +741,26 @@ def contourf(x, y, z, ax, **kwargs):
     return primitive
 
 
+def _is_monotonic(coord, axis=0):
+    """
+    >>> _is_monotonic(np.array([0, 1, 2]))
+    True
+    >>> _is_monotonic(np.array([2, 1, 0]))
+    True
+    >>> _is_monotonic(np.array([0, 2, 1]))
+    False
+    """
+    if coord.shape[axis] < 3:
+        return True
+    else:
+        n = coord.shape[axis]
+        delta_pos = (coord.take(np.arange(1, n), axis=axis)
+                     >= coord.take(np.arange(0, n-1), axis=axis))
```

Comment:
To make flake8 happy, the operator should be on the first line, i.e.:

```python
delta_pos = (coord.take(np.arange(1, n), axis=axis) >= 
             coord.take(np.arange(0, n-1), axis=axis))
```

Reply:
ok. my last commit should fix this. sry, i can't seem to get my flake8 to do the same things as travis.